src/ticks/debugger_gdb_packets.c

```diff
@@ -86,7 +94,13 @@ void write_flush(sock_t sockfd)
     while (write_index < out.end)
     {
         ssize_t nwritten;
+#ifndef WIN32
```

Comment:
The project I used it in has dropped off the internet, but I used a set of macros in it which sorted out on the winsock differences.

There's a gist here with the essence: https://gist.github.com/suborb/cc1a4828ae30247e21fe5064f23f44a2